eng/mgmt/mgmtmetadata/network_resource-manager.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: latest
```

Comment:
Looks like you changed the permissions on eng/AzSdkTestLibKey.snk and eng/Packages.Data.props as well can you please undo those changes?

Reply:
As suggested by Erich in another comment, I will close this PR and submit a new one with changes to Microsoft.Azure.Management.Network to speed things up.